dotnet/src/Functions/Functions.OpenApi/Model/RestApiOperationPayloadProperty.cs

```diff
@@ -39,28 +39,37 @@ public sealed class RestApiOperationPayloadProperty
     /// </summary>
     public KernelJsonSchema? Schema { get; }
 
+    /// <summary>
+    /// The default value.
+    /// </summary>
+    public object? DefaultValue { get; }
+
     /// <summary>
     /// Creates an instance of a <see cref="RestApiOperationPayloadProperty"/> class.
     /// </summary>
-    /// <param name="name">Property name.</param>
-    /// <param name="type">Property type.</param>
-    /// <param name="isRequired">Flag specifying if the property is required or not.</param>
-    /// <param name="properties">Properties.</param>
-    /// <param name="description">Property description.</param>
+    /// <param name="name">The name of the property.</param>
+    /// <param name="type">The type of the property.</param>
+    /// <param name="isRequired">A flag specifying if the property is required or not.</param>
+    /// <param name="properties">A list of properties for the payload property.</param>
+    /// <param name="description">A description of the property.</param>
     /// <param name="schema">The schema of the payload property.</param>
+    /// <param name="defaultValue">The default value of the property.</param>
+    /// <returns>Returns a new instance of the <see cref="RestApiOperationPayloadProperty"/> class.</returns>
     public RestApiOperationPayloadProperty(
         string name,
         string type,
         bool isRequired,
         IList<RestApiOperationPayloadProperty> properties,
```

Comment:
Does this has a strong reason to be a list or could this be a ICollection or Enumerable?

Reply:
No specific reason, looking at implementation either of the suggested types would work. I'll keep it as is until the moment we have strong reasons to change.